code/__HELPERS/mobs.dm

```diff
@@ -312,3 +312,9 @@ proc/add_ghostlogs(var/mob/user, var/obj/target, var/what_done, var/admin=1, var
 			L.Add(A)
 
 	return L
+
+/proc/is_implanted(mob/M)
```

Comment:
maybe make it a mob proc?
maybe make it take the type of the implant you're interested in?
alternatively rename it to is_loyalty_implanted